### ISSUE TITLE
Design improvements

### ISSUE THREAD

**Issue by phlind** [SkepticMystic/graph-analysis#17]
I would propose to make the plugin look less cluttered by moving the algorithm setting and the "zero" and "infinity" checkboxes into the settings panel. 

Also, in the link predictions panel the table header should not be shown when there's no note
![Bildschirmfoto 2021-11-02 um 13 49 32](https://user-images.githubusercontent.com/30386310/139849702-08b40c4c-a7ff-4209-a220-1f70d7fa1b74.png)
.

**Comment by SkepticMystic** [comment]
What do you mean into the settings panel? The plugin settings? I feel that's quite an inconvenience if you want to quickly change algs. Maybe there is a better middle ground.

And yes, I agree that the table header should be hidden if there are no notes

**Comment by SkepticMystic** [comment]
@phlind You'll see I've slimmed down the AnalysisView a bit. Let me know your thoughts

**Comment by SkepticMystic** [comment]
@phlind lots of new changes to the ui in 0.6.4! Let us know what you think

**Comment by phlind** [comment]
Yes, I see many changes going on in the last two week, and I really like it, especially the scrolling mode. ❤️ 

Maybe one improvement that could be made is to make the font size the same as the file explorer. I just checked with the default theme and no snippets. Here the font looks bigger than the actual note text:

![Bildschirmfoto 2021-11-15 um 08 18 16](https://user-images.githubusercontent.com/30386310/141738476-0dc07392-6bc5-402b-881d-78d3fed94334.png)

Of course that's very easy to achive via CSS by every user, but maybe one could make it default?
